test/images/agnhost/netexec/netexec.go

```diff
@@ -213,14 +214,15 @@ func main(cmd *cobra.Command, args []string) {
 	}
 }
 
-func addRoutes(mux *http.ServeMux, exitCh chan shutdownRequest) {
+func addRoutes(mux *http.ServeMux, termCh chan os.Signal, exitCh chan shutdownRequest) {
 	mux.HandleFunc("/", rootHandler)
 	mux.HandleFunc("/clientip", clientIPHandler)
 	mux.HandleFunc("/header", headerHandler)
 	mux.HandleFunc("/dial", dialHandler)
 	mux.HandleFunc("/echo", echoHandler)
 	mux.HandleFunc("/exit", func(w http.ResponseWriter, req *http.Request) { exitHandler(w, req, exitCh) })
 	mux.HandleFunc("/healthz", healthzHandler)
+	mux.HandleFunc("/readyz", readyzHandler(termCh))
```

Comment:
can you add the description of the new handler? , L93 per example
https://github.com/kubernetes/kubernetes/pull/110174/files#diff-f26ab21963f05c932bece947c19bcff504d1f6a4d53e4e33f2029908126bd12aR93